examples/tutorials/08_research_tutorial/1_Research_Introduction.ipynb

```diff
@@ -0,0 +1,2513 @@
+{
```

Comment:
Name folder as `examples/tutorials/research/` and numerate files like `01_introduction_into_research`, `02_basic_research`, etc

Reply:
@a-arefina !